pages/blocks/[id].tsx

```diff
@@ -50,6 +50,20 @@ const BLOCK_CARDS = [
     },
     icon: <DifficultyIcon />,
   },
+  {
+    key: 'prev-hash-card',
+    label: 'Previous Block hash',
+    value: block => {
+      const previous_block_hash = safeProp('previous_block_hash')(block)
+      return (
+        <CopyValueToClipboard
+          value={previous_block_hash}
```

Comment:
Could this link to the block? Similar to the hash of each transaction linking to the transaction page

Reply:
Added!